endpoints/openrtb2/auction.go

```diff
@@ -2492,19 +2492,18 @@ func checkIfAppRequest(request []byte) (bool, error) {
 	return false, nil
 }
 
-func validateStoredBidRespAndImpExtBidders(bidderExts map[string]json.RawMessage, storedBidResp stored_responses.ImpBidderStoredResp, impId string) error {
+func (deps *endpointDeps) validateStoredBidRespAndImpExtBidders(bidderExts map[string]json.RawMessage, storedBidResp stored_responses.ImpBidderStoredResp, impId string) error {
 	if bidResponses, ok := storedBidResp[impId]; ok {
 		if len(bidResponses) != len(bidderExts) {
 			return generateStoredBidResponseValidationError(impId)
 		}
 
 		for bidderName := range bidResponses {
-			bidder := bidderName
-			normalizedCoreBidder, ok := openrtb_ext.NormalizeBidderName(bidder)
-			if ok {
-				bidder = normalizedCoreBidder.String()
+			_, bidderNameOk := deps.normalizeBidderName(bidderName)
+			if !bidderNameOk {
+				return fmt.Errorf(`unrecognized bidder "%v"`, bidderName)
 			}
-			if _, present := bidderExts[bidder]; !present {
+			if _, present := bidderExts[bidderName]; !present {
```

Comment:
Mostly for my education. Why are we not using normalised bidder name to lookup on `bidderExts`.  
Here `bidderExts` is prepared from core bidder names. So should use normalised bidder name

Reply:
I added a description with explanations how it works. We need to use bidder name as it it specified in imp, not normalized bidder name.

Reply:
![Screenshot 2023-10-11 at 3 41 07 PM](https://github.com/prebid/prebid-server/assets/4790765/9f8658ae-8580-4a5a-aeaf-e36aec1dc964)